lib/controllers/get-token.js

```diff
@@ -21,8 +21,14 @@ module.exports = function (req, res) {
   var logger = req.app.get('stormpathLogger');
 
   function writeErrorResponse(err) {
+    var error = {
```

Comment:
Please see my comment about that: https://github.com/stormpath/express-stormpath/pull/340#issuecomment-187131588